royal/ext/sqla.py

```diff
@@ -0,0 +1,168 @@
+import logging
+
+from pyramid.location import lineage
+import royal
+from sqlalchemy.orm.collections import MappedCollection
+
+log = logging.getLogger(__name__)
+
+
+def includeme(config):
+    config.scan(__name__)
+
+
+class Collection(royal.Collection):
+    sa_model = None
+    sa_exceptions = None
+    entity_cls = None
```

Comment:
Yeah, that's what I did at first.. But why only for `sa_model` ? what about `entity_cls` ?
May be we just document the expected injection ? 
I used to be a fan of interfaces back in the Java and C++ days, but it doesn't sound like a python thing..